lib/collection/request-auth/oauth1.js

```diff
@@ -103,16 +105,14 @@ module.exports = {
                 system: true
             });
         }
+        else if (_.isString(request.method) && request.method.match(MATCH_PUTPOST) &&
+                (request.body && request.body.mode === RequestBody.MODES.urlencoded)) {
+            _.forEach(signatureParams, function (param) {
+                request.body.urlencoded.add(param);
+            });
+        }
         else {
-            if (/PUT|POST/.test(request.method) &&
```

Comment:
Reinstate this code and ensure that the only changes in this file are space removals inside objects.